docs/sql-guide/functions/function-flatten.md

```diff
@@ -0,0 +1,76 @@
+---
+title: FLATTEN()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# FLATTEN() function
+
+The `FLATTEN()` function is used when a query wants to get distinct or group on individual members of [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) columns
+
+## Syntax
+
+```
+flatten(column)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required? | Further information |
+|---|---|---|---|---|
+| `column` | IDSET/STRINGSET | [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) columns | Yes | This should only be used with `GROUP BY` queries |
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `ID`/`STRING` | individual values of passed column |
+
+## Additional information
+
+* This can only be used in the [WITH clause](/docs/sql-guide/statements/statement-select/#with_clause)
```

Comment:
The `flatten` function can be used in:
* [`SELECT...WITH`](/docs/sql-guide/statements/statement-select/#with_clause) queries
* `GROUP BY` queries